src/Microsoft.VisualStudio.AppDesigner/Common/Utils.vb

```diff
@@ -664,6 +664,21 @@ Namespace Microsoft.VisualStudio.Editors.AppDesCommon
 
                 TelemetryService.DefaultSession.PostEvent(userTask)
             End Sub
+
+            Private Const BinaryFormatterEventName As String = "vs/projectsystem/appdesigner/binaryformatter"
+            Public Enum BinaryFormatterType
+                Serialize = 0
+                Deserialize = 1
+            End Enum
+
+            Public Shared Sub LogBinaryFormatterEvent(functionName As String, className As String, type As BinaryFormatterType)
+                Dim userTask = New UserTaskEvent(BinaryFormatterEventName, TelemetryResult.Success)
+                userTask.Properties("vs.projectsystem.appdesigner.binaryformatter.functionname") = functionName
```

Comment:
would it be better to use a constant string prefix like the TelemetryPropertyName class does?

Reply:
Sure. I can do that.

Reply:
Updated both TelemetryLogger classes to use this pattern for all logging functions.